### PR TITLE
Mail services: update dovecot2, dovecot2-sieve, rspamd, apache-solr8

### PR DESCRIPTION
**Pull request by essandess**
Mail services: update dovecot2, dovecot2-sieve, rspamd, apache-solr8

* java/apache-solr8/Portfile
* mail/dovecot2-sieve/Portfile
* mail/dovecot2/Portfile
* mail/rspamd/Portfile

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by reneeotten** [comment]
please follow the Commit Message Guidelines linked in the PR checklist that you checked off, in particular the one about "be specific".

File: textproc/libstemmer/Portfile [hunk 1]
```diff
@@ -3,31 +3,31 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        snowballstem snowball 2a3f28097af3651f0c1c6da8998d676b487d5651
+github.setup        snowballstem snowball 2.0.0 v
```
Comment (mf2k) [inline textproc/libstemmer/Portfile]: I'm guessing that this will require invoking the `epoch` line.

Comment (reneeotten) [reply textproc/libstemmer/Portfile]: probably not as the version was previously set to ```0.1-20191013``` (see line 13 below)

Comment (mf2k) [reply textproc/libstemmer/Portfile]: Oh yes you are right. I didn't see it because the version field doesn't usually appear there.

**Pull request merged**